src/components/AddView.js

```diff
@@ -1,10 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { db } from '../lib/firebase';
 
 const AddView = () => {
+  const [state, setState] = useState({
+    purchaseItem: '',
+    howSoon: '7',
+    lastPurchaseDate: null,
+  });
+
+  const handleChange = (e) => {
+    setState({
+      ...state,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (state.purchaseItem.trim()) {
+      let token =
+        localStorage.length !== 0 ? localStorage.getItem('token') : 'LA';
+      db.collection(token)
+        .add({
+          purchaseItem: state.purchaseItem,
+          howSoon: state.howSoon,
+          lastPurchaseDate: state.lastPurchaseDate,
+        })
+        .then(() => {
+          alert('Purchase Item has been recorded');
```

Comment:
Also, I think it would be better if, instead of using an alert, you displayed the notification on the page instead, and then the user could click a button to dismiss it. I think that would make for better UX.

Reply:
Hey @C-Zekeri, this is a great idea. But this will need us to create a notification component which is out of scope of this task. Let me know what you think

Reply:
Yes, that's true, it appears it's out of the scope.